api/internal/worker/handlers/invhandlers/utils.go

```diff
@@ -109,6 +109,7 @@ func isSet(resourceID *string) bool {
 
 // parsePagination parses the pagination fields converting them to limit and offset for the inventory APIs.
 func parsePagination(pageSize, off *int) (limit, offset uint32, err error) {
+	limit, offset = 20, 0
```

Comment:
what is this 20?

Reply:
The OAPI definition has a default, I would expect that to be pre-populated on the field.

Reply:
It was not pre-populated. Now it is, see the fixed host_test.go file in the PR.